src/core/client/stream/tabs/Comments/Comment/ReplyCommentForm/CreateCommentReplyMutation.ts

```diff
@@ -185,7 +185,7 @@ function commit(
               createdAt: viewer.createdAt,
               ignoreable: false,
               status: {
-                current: [GQLUSER_STATUS.ACTIVE],
+                current: [],
```

Comment:
Not sure if you are running into this: But be aware that Relay produces some false warnings – https://github.com/facebook/relay/pull/2760

Reply:
It will show a _false_ warning when putting scalar values inside an array in the optimistic response. Sth like 

```
Warning: validateMutation: `optimisticResponse` for mutation `ChangeTagsMutation`, contains an unused field ROOT.changeTags.tags.0
```

Reply:
I was indeed trying to silence these errors. I will put these back, as I don't want to be screwing with the optimistic response more than I have to. Even though they do spam the log with warnings.